lib/gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -34,7 +34,9 @@ def self.runcmd(cmd_str)
     # If sudo is required, ensure you have /etc/sudoers.d/miq
     # Cmnd_Alias MOUNTALL = /bin/mount, /bin/umount
     # %wheel ALL = NOPASSWD: MOUNTALL
-    rv = `sudo #{cmd_str} 2>&1` if rv.include?("mount: only root can do that")
+    if rv.include?("mount: only root can do that") || rv.include?('umount failed: Operation not permitted')
```

Comment:
Can we make this less dependent on the error message, that seems fragile to me.

Maybe this is more exact?

```
if cmd_str =~ /^(mount|umount)/ && $CHILD_STATUS.exit == 1
```